static/.htaccess

```
@@ -8,10 +8,12 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+# Remove OSS section
+RewriteRule ^docs/components/zeebe/open-source(.*)$ /docs/components/zeebe/zeebe-overview/$1 [R=301,L]
```

```suggestion
RewriteRule ^docs/components/zeebe/open-source(.*)$ /docs/components/zeebe/zeebe-overview/ [R=301,L]
```

@akeller I suspect the failing links are because the part after `open-source/` is being preserved in the target URL. 

So for example, the original URL `docs/components/zeebe/open-source/community-contributions/` gets rewritten as `/docs/components/zeebe/zeebe-overview/community-contributions/`, which doesn't exist. The error message in the build failure isn't totally clear on that, so I'm kind of guessing...

This suggestion would redirect all child pages to the overview page itself, dropping the individual page's slug.

---

THANK YOU!